web-ui/install-deps.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+cd /home/frankbria/projects/codeframe/web-ui
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)